storage/tests/integration/test_upload_resumable.py

```diff
@@ -0,0 +1,32 @@
+import os
+import uuid
+import json
+import aiohttp
+import pytest
+
+from gcloud.aio.storage import Storage
+
+PROJECT = os.environ['GCLOUD_PROJECT']
+CREDS = os.environ['GOOGLE_APPLICATION_CREDENTIALS']
+BUCKET_NAME = os.environ['BUCKET_NAME']
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize('data_to_upload,expected_data,content_type', [
+    ('test', 'test', 'text/plain'),
+    (json.dumps({'data': 1}), '{"data": 1}', 'application/json'),
+    (json.dumps([1, 2, 3]), json.dumps([1, 2, 3]), 'application/json'),
+    ('test'.encode('utf-8'), 'test'.encode('utf-8'), 'application/octet-stream'),
+])
+async def test_given_dict_when_upload_resumable_then_check_upload(data_to_upload, expected_data, content_type):
```

Comment:
Please format your code with an 80 character limit. For some reason, we had accidentally disabled tests for external contributors -- I've re-enabled that, so you should see linting results next time you push.

Reply:
You can follow the instructions [here](https://github.com/talkiq/gcloud-aio/blob/master/.github/CONTRIBUTING.rst#coding-conventions) to run `pre-commit` locally.